middleman_protocol/middleman_protocol/helpers.py

```diff
@@ -0,0 +1,81 @@
+import socket
+
+from mypy.types import Iterable
+
+from middleman_protocol.constants import ESCAPE_CHARACTER
+from middleman_protocol.constants import ESCAPE_SEQUENCES
+from middleman_protocol.constants import FRAME_PAYLOAD_LENGTH_STARTING_BYTE
+from middleman_protocol.constants import FRAME_PAYLOAD_STARTING_BYTE
+from middleman_protocol.constants import FRAME_SEPARATOR
+from middleman_protocol.constants import FRAME_SEPARATOR_LENGTH
+from middleman_protocol.constants import RECEIVE_BYTES_PER_LOOP
+from middleman_protocol.message import AbstractMiddlemanMessage
+
+
+def handle_message_receive(connection: socket.socket, encoding_expected: bool=False) -> Iterable[bytes]:
```

Comment:
It has to have `handle_` in its name, so I am choosing first option. :+1: done

Reply:
Why? A handler sounds like a callback. Or at least like a function that does not have a specific thing it does but rather just "handles" some event - where the things needed to handle it may not be well defined or may change over time. But this is a generator. It has a very specific purpose - to read a frame. It does not handle the frame, i.e. does not do anything with it. It's a utility that the application would use before passing the returned data to the actual handler.

`middleman_protocol` is a library and for me a handler clearly sounds like something supplied by the application, not like something that the library itself does.

Reply:
Names changed according to the specs. :+1: done